robottelo/utils/issue_handlers/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Methods related to issue handlers in general
+from robottelo.utils.issue_handlers import bugzilla
+
+
+handler_methods = {'BZ': bugzilla.is_open_bz}
+SUPPORTED_HANDLERS = tuple(f"{handler}:" for handler in handler_methods.keys())
+
+
+def is_open(issue, data=None):
+    """Check if specific issue is open.
+
+    Issue must be prefixed by its handler e.g:
+
+    Bugzilla: BZ:123456
+
+    Arguments:
+        issue {str} -- A string containing handler + number e.g: BZ:123465
+        data {dict} -- Issue data indexed by <handler>:<number> or None
+    """
+    # Handlers can be extended to support different issue trackers.
+    if str(issue).startswith(SUPPORTED_HANDLERS):
+        handler_code, _, _ = str(issue).partition(":")
```

Comment:
```suggestion
        handler_code, *_ = str(issue).partition(":")
```

Reply:
Again just moved the original function here, but I'll specifically index.